sdk/src/constants/object.ts

```diff
@@ -1,20 +1,19 @@
 import { COIN, ScableCoin, SharedObjectRef } from '@/types';
 
+export const CORE_PACKAGE_ID = '0x0b6ba9889bb71abc5fa89e4ad5db12e63bc331dba858019dd8d701bc91184d79';
+export const STRAP_FOUNTAIN_PACKAGE_ID = '0x204f5f18b9b4d10eddc0f7256284ed3c655bd06e9425801e60b262152d47de50';
 export const BUCKET_OPERATIONS_PACKAGE_ID: string =
-  '0x89a9f07d87b37c2a5d2144bdb5f7f606324cc38cc923e6c75543a053ca54c8db';
+  '0x50e0e9e549574c2ab7ee24d37bffd38eaeb2e93e83bcac26aac081606ac68fae';
```

Comment:
@JustaLiang thats weird, when I install the version `"bucket-protocol-sdk": "0.10.39",` from npm, I still get the old package id `0x89a9f07d87b37c2a5d2144bdb5f7f606324cc38cc923e6c75543a053ca54c8db` - which is wrong.

<img width="1230" alt="image" src="https://github.com/user-attachments/assets/d5474151-1e9e-49d4-8c55-6e4249f52f31" />

Can you check this please, or publish again to npm?

Thanks a lot!